lib/onc_certification_g10_test_kit/bulk_export_validation_tester.rb

```diff
@@ -166,7 +170,13 @@ def check_file_request(url) # rubocop:disable Metrics/CyclomaticComplexity
         scratch[:patient_ids_seen] = patient_ids_seen | [resource.id] if resource_type == 'Patient'
 
         unless resource_is_valid?(resource: resource, profile_url: profile_url)
-          assert false, "Resource at line \"#{line_count}\" does not conform to profile \"#{profile_url}\"."
+          if first_error.key?(:line_number)
+            @invalid_line_count += 1
```

Comment:
`@invalid_resource_count` would make more sense to me.

Reply:
Haha. I had the name `@invalid_resource_count ` but changed to `@invalid_line_count` in the last minute. I will change it back